test/end-to-end-tests/src/usecases/signup.js

```diff
@@ -32,6 +32,7 @@ module.exports = async function signup(session, username, password, homeserver)
         await nextButton.click();
     }
     //fill out form
+    await session.delay(100);
```

Comment:
whenever we `delay` in a test, we generally try to comment on it so we know when we can remove that delay